### PR TITLE
NetSuite recipe migration

### PR DESCRIPTION
**Pull request by bennettgo**

### CODE REVIEW

File: docs/connectors/netsuite.md [hunk 1]
```diff
@@ -218,6 +218,10 @@ When linking an object to another object via Workato (e.g. when creating a trans
 
 The recipe is able to retrieve the custom fields of your selected NetSuite object for you to read from or write to. All custom fields are grouped in Workato.
 
+::: tip 19/5/2020 Enhancement
+As of 19/5/2020, we have made a significant enhancement to the migration of NetSuite recipes from Sandbox to Production environments. This requires enhancement requires steps to be take for any NetSuite recipes created on or before 19/5/2020. Learn more about from our migration guide [here](/connectors/netsuite/recipe-migration-guide.md)
```
Comment (eeshansim) [inline docs/connectors/netsuite.md:222]: As of 19/5/2020, we have made a significant enhancement to the migration of NetSuite recipes from Sandbox to Production environments. This ~requires enhancement~ requires certain steps to be ~take~ taken for any NetSuite recipes created on or before 19/5/2020. Learn more about from our migration guide [here](/connectors/netsuite/recipe-migration-guide.md)

File: docs/connectors/netsuite/recipe-migration-guide.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## For recipes created after 19/5/2020
+
+No action is needed. When switching connections, all mappings will be preserved.
+
+## For recipes created before 19/5/2020
+
+You will need to do an initial remapping of custom fields **only if you still plan to change the NetSuite connection for this recipe.**
+
+### Steps needed
+1. For a specific recipe, you will need to go to each action and performa remapping of datapills to custom fields. In the example below, i have an `Update Purchase order` action with a single custom field selected.
```
Comment (eeshansim) [inline docs/connectors/netsuite/recipe-migration-guide.md:19]: For a specific recipe, you will need to go to each action and ~performa~ perform remapping of datapills to custom fields. In the example below, ~i~ we have an `Update Purchase order` action with a single custom field selected.

File: docs/connectors/netsuite/recipe-migration-guide.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+![NetSuite custom fields](~@img/connectors/netsuite/recipe-migr-1.png)
+*Update Purchase Order with 1 Custom field*
+
+![NetSuite custom fields](~@img/connectors/netsuite/recipe-migr-2.png)
+*Custom field mapping*
+
+2. To enable seamless migration, you will need to go to the `Standard record` selection and reselect the `Purchase Order` object. When searching, you should see 2 `Purchase Order` options. **You should select the top most option.** After selecting this, your custom field selection will be cleared out. Be sure to either have a reference of this recipe in another browser tab open or a copy of this recipe elsewhere.
```
Comment (eeshansim) [inline docs/connectors/netsuite/recipe-migration-guide.md:27]: This happens after switching connection right?

File: docs/connectors/netsuite/recipe-migration-guide.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+![NetSuite custom fields](~@img/connectors/netsuite/recipe-migr-3.png)
+*Custom field mapping*
+
+3. Reselect your custom fields needed and remap all custom fields again. Once this is done, you have now enabled this new feature on this action. You will need to continue this for the rest of the actions in this recipe and other recipes where needed.
```
Comment (eeshansim) [inline docs/connectors/netsuite/recipe-migration-guide.md:32]: Reselect ~your~ the custom fields you ~needed~ need and remap all custom fields again

File: docs/connectors/netsuite/recipe-migration-guide.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+## FAQ
+1. Do i need to migrate every action and every recipe?
+
+There is no requirement that you need to migrate all existing recipes and/or actions. You can choose to only do this for selected recipes and actions where custom fields are used **AND** this recipe is going to be migrated via recipe lifecycle management to a production Workato workspace. For actions and recipes that only interact with standard fields, this enablement is not necessary.
```
Comment (eeshansim) [inline docs/connectors/netsuite/recipe-migration-guide.md:37]: Added "if"
.. are used **AND** if this recipe is going...

File: docs/connectors/netsuite/recipe-migration-guide.md (hunk 3 above)
Comment (bennettgo) [reply docs/connectors/netsuite/recipe-migration-guide.md:27]: Nope, this is before even switching connection.

File: docs/connectors/netsuite.md [hunk 6]
```diff
@@ -218,6 +218,10 @@ When linking an object to another object via Workato (e.g. when creating a trans
 
 The recipe is able to retrieve the custom fields of your selected NetSuite object for you to read from or write to. All custom fields are grouped in Workato.
 
+::: tip 19/5/2020 Enhancement
+As of 19/5/2020, we have made a significant enhancement to the migration of NetSuite recipes from Sandbox to Production environments. This requires certain steps to be take taken for any NetSuite recipes created on or before 19/5/2020. Learn more about from our migration guide [here](/connectors/netsuite/recipe-migration-guide.md)
```
Comment (eeshansim) [inline docs/connectors/netsuite.md:222]: This requires certain steps to be ~take~ taken for

**Pull request merged**